src/Containers/CadView.jsx

```diff
@@ -429,7 +430,7 @@ export default function CadView({
     resetState()
     const repoFilePath = modelPath.gitpath ? modelPath.getRepoPath() : modelPath.filepath
     window.removeEventListener('beforeunload', handleBeforeUnload)
-    navigate(`${pathPrefix}${repoFilePath}`)
+    navWith(navigate, `${pathPrefix}${repoFilePath}`)
```

Comment:
we might want to clear both hash and query paramters here?

Reply:
Sure, going to try that and removing search as well.